reports/opendmarc-expire.8.in

```diff
@@ -56,7 +56,7 @@ This man page covers the version of
 that shipped with version @VERSION@ of
 .I OpenDMARC.
 .SH COPYRIGHT
-Copyright (c) 2012, 2014, The Trusted Domain Project.  All rights reserved.
+Copyright (c) 2012, 2014, 2018, The Trusted Domain Project.  All rights reserved.
```

Comment:
Why?   Nothing changed here.

Reply:
Does the copyright reflect the man page itself or the project? I would think the latter as it should be a clue to the user as to when the last changes were applied.